lib/internal/url.js

```diff
@@ -701,7 +705,7 @@ class URLSearchParams {
   }
 
   [util.inspect.custom](recurseTimes, ctx) {
-    if (!this || !(this instanceof URLSearchParams)) {
+    if (!this || this[searchParamsInstance] !== true) {
```

Comment:
I'm still reluctant about writing out `this[searchParamsInstance] !== true` when a `!this[searchParamsInstance]` would do. Is this change visible performance-wise?